### PR TITLE
Error when the dashjs player errors

### PR DESCRIPTION
**Pull request by forbesjo**

### CODE REVIEW

File: test/dashjs.test.js [hunk 1]
```diff
@@ -92,12 +92,16 @@
...
-              }
+              },
+
+              on: function() {}
             };
           }
         };
       };
 
+      dashjs.MediaPlayer.events = { ERROR: '' };
```
Comment (gesinger) [inline test/dashjs.test.js]: Is this being used anywhere?

Comment (forbesjo) [inline test/dashjs.test.js]: `ERROR` itself doesn't need to exist for the tests but `dashjs.MediaPlayer.events` does

File: src/js/videojs-dash.js [hunk 2]
```diff
@@ -80,6 +80,13 @@ class Html5DashJS {
       this.mediaPlayer_.setLimitBitrateByPortal(false);
     }
 
+    this.mediaPlayer_.on(dashjs.MediaPlayer.events.ERROR, function(event) {
+      this.tech_.error({
```
Comment (gesinger) [inline src/js/videojs-dash.js]: Do we want to error on all dash.js errors (including timed text issues, download errors, etc.)?

Comment (forbesjo) [inline src/js/videojs-dash.js]: - [ ] whitelist errors we want videojs to error on

**Comment by forbesjo** [comment]
Closing in favor of #224

**Pull request closed without merging**